### PR TITLE
Fix #9553 "Open Recent" not disabled

### LINKED ISSUE #9553: [MU4 Issue] File->Open Recent not disabled even if there is no recent files.
... and does nothing when clicked

see https://youtu.be/0Y0X10yiOYg?t=666

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: https://github.com/musescore/MuseScore/issues/9553

This commit solved the bug that File->"Open Recent" is not disabled even if
there is no recent files. Now, it decides whether to enable "Open
Recent" on the basis of whether the recent scores list is empty.

Also, I have removed the comment: need implement of export since I think it has been implemented. Please tell me otherwise.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by RobFog** [comment]
Does the second commit belong in this pull request?

**Comment by cbjeukendrup** [comment]
Because both commits are about improvements to the actions and  menus, I think it makes sense to group them. (Otherwise, you would end up with a lot of small PRs...)

File: src/notation/internal/notationuiactions.cpp [hunk 1]
```diff
@@ -213,7 +214,7 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-visible",
              mu::context::UiCtxNotationOpened,
              QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
-             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements")
+             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
```
Comment (cbjeukendrup) [inline src/notation/internal/notationuiactions.cpp:217]: You've an extra comma here, that's why the test builds are failing! Please amend your commit and force-push, then the checks will be run again and everything will be all right.

For the rest, it looks great!

Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:217]: That comma is not needed. Actually the whole line is not needed (and then the comma on the line abobe is not needed anomre either)

File: src/notation/internal/notationuiactions.cpp [hunk 2]
```diff
@@ -184,7 +184,8 @@ const UiActionList NotationUiActions::m_actions = {
              ),
     UiAction("copy",
              mu::context::UiCtxNotationFocused,
-             QT_TRANSLATE_NOOP("action", "Copy")
+             QT_TRANSLATE_NOOP("action", "Copy"),
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:187]: Here I believe you'd need to duplicate that line

File: src/notation/internal/notationuiactions.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:217]: Sure this comma is the culprit?

Comment (cbjeukendrup) [reply src/notation/internal/notationuiactions.cpp:217]: Yes I am. Strange discrepancy in C++: in an enum or list, a comma before a `}` is allowed, but before a `)` it gives an error.

Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:217]: OK, so that's 2 build problems then, the other one needs to get fixed too

**Comment by HemantAntony** [comment]
I knew that it would bring an error. I saw it in the last second, but I was too tired to fix it. Will do that soon!

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**